src/integ_test_resources/ios/sdk/integration/cdk/custom_resources/iot_custom_authorizer_provider/README.md

```diff
@@ -0,0 +1,14 @@
+== IoT Custom Authorizer
+
```

Comment:
Thank you for setting this as a precedent! Would be great if we can get a `README.md` for each new stack, as they come in.

Reply:
> Could you add a note about that?

As noted above, I'll migrate to using KMS, so this will be removed in an upcoming PR

Reply:
Removed hardcoded keypair in favor of KMS custom resource